tools/pot/openvino/tools/pot/algorithms/quantization/minmax/algorithm.py

```diff
@@ -7,7 +7,7 @@
 import numpy as np
 
 from openvino.tools.pot.graph import save_model
-from ..utils import get_tensor_statistics, get_stat_name_by_config
+from ..utils import get_tensor_statistics, get_stat_name_by_config, configs_creation, proxy_metrics
```

Comment:
I think that these changes (and others below) are messed up with another branch, isn't it, @MilanaShhanukova?